src/main/daml/Daml/Finance/Interface/Types/Fpml.daml

```diff
@@ -248,3 +249,21 @@ data FixedRateSchedule = FixedRateSchedule with
     --   conformant document.
   -- type_ : Optional SpreadScheduleType
```

Comment:
remove comment?

Reply:
This is an imported data structure from the lib message integration repo:
https://github.com/digital-asset/lib-message-integration
I have not yet implemented all FpML features though (and not sure if we will), so I commented out some fields for now.